### PR TITLE
feat: add asr completion spec

### LINKED ISSUE #945: [asr] missing spec
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue
- [ ] I would like to work on this.

### What CLI tool does this relate to?

asr

### Which statement makes the most sense?

This completion spec does not exist

### Issue Details

See https://keith.github.io/xcode-man-pages/asr.8.html

### PR DESCRIPTION
**Pull request by SerpentBytes**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Fixes #945 by adding completion spec for `asr`

**What is the current behavior? (You can also link to an open issue here)**
Missing completion spec for `asr`. See #945.

**What is the new behavior (if this is a feature change)?**
Completion spec for `asr` are added

**Additional info:**
- Deprecated options were not included
- See `man` page references here: https://keith.github.io/xcode-man-pages/asr.8.html

### CODE REVIEW

**Comment by fedeci** [comment]
recheck

**Comment by SerpentBytes** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by SerpentBytes** [comment]
@fedeci, To clarify: My name is Taimoor Dawami; however my GitHub username is SerpentBytes

**Comment by SerpentBytes** [comment]
recheck

**Comment by SerpentBytes** [comment]
recheck

**Comment by SerpentBytes** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/asr.ts [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+            name: "interface",
+            description: "The network interface to be used for multicasting",
+          },
+        },
+        {
+          name: "--config",
+          description: "Server requires a configuration file to be passed",
+          args: {
+            name: "configuration",
+            description: "Configuration file in standard property list format",
```
Comment (fedeci) [inline src/asr.ts:212]: Here you can use the `filepaths` template function to provide file suggestion with the correct extensions. (https://fig.io/docs/reference/generator/prebuilt-generators#filepathsoptions)

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for the PR @SerpentBytes! I left one suggestion to address.

File: src/asr.ts (hunk 1 above)
Comment (SerpentBytes) [reply src/asr.ts:212]: Great catch! working on it 👍🏼

Comment (SerpentBytes) [reply src/asr.ts:215]: @fedeci, I resolved this in [e51566b](https://github.com/withfig/autocomplete/pull/1711/commits/e51566bda2f664b82d04a270cb64a3be7f0bbe37).

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**